Modules/ArcGIS/Configuration/ArcGISConfigure.ps1

```diff
@@ -524,271 +493,36 @@ Configuration ArcGISConfigure
                             RegisterWebAdaptorForCName = $False
-                            CertificateFileLocation = $Node.SslCertifcate.Path
-                            CertificatePassword = $Node.SslCertifcate.Password
+                            CertificateFileLocation = $SSLCertificate.Path
+                            CertificatePassword = $SSLCertificate.Password
                             EnableSSL = $True
                             SslRootOrIntermediate = $SslRootOrIntermediate
                             DependsOn =  $Depends
                         }
-                    }elseif((($AllNodes | Where-Object { ($_.Role -icontains 'LoadBalancer') -or ($_.Role -icontains 'ServerWebAdaptor') }  | Measure-Object).Count -eq 0) -and $ConfigurationData.ConfigData.Server.SslCertifcate.Path){
-                        ArcGIS_Server_TLS "Server_TLS_$($Node.NodeName)"
-                        {
-                            Ensure = 'Present'
-                            SiteName = 'arcgis'
-                            SiteAdministrator = $PSACredential                         
-                            CName = $ConfigurationData.ConfigData.Server.SslCertifcate.Alias
-                            RegisterWebAdaptorForCName = $False
-                            CertificateFileLocation = $ConfigurationData.ConfigData.Server.SslCertifcate.Path
-                            CertificatePassword = $ConfigurationData.ConfigData.Server.SslCertifcate.Password
-                            EnableSSL = $True
-                            SslRootOrIntermediate = $SslRootOrIntermediate
-                            DependsOn =  $Depends
-                        } 
-                    }else{
-                        ArcGIS_Server_TLS "Server_TLS_$($Node.NodeName)"
-                        {
-                            Ensure = 'Present'
-                            SiteName = 'arcgis'
-                            SiteAdministrator = $PSACredential                         
-                            CName = $MachineFQDN
-                            RegisterWebAdaptorForCName = $False
-                            EnableSSL = $True
-                            DependsOn =  $Depends
-                        } 
                     }
-                    #Check if to add fallbacks in case per node cert is not given or missing.
-
-                    if($Federation -and ($Node.NodeName -ieq $PrimaryServerMachine) -and (($AllNodes | Where-Object { ($_.Role -icontains 'LoadBalancer' -or $_.Role -icontains 'ServerWebAdaptor')}  | Measure-Object).Count -eq 0))
-                    {
-                        if(($AllNodes | Where-Object { $_.Role -icontains 'PortalWebAdaptor'}  | Measure-Object).Count -eq 0)
-                        {
-                            if($PrimaryPortalMachine)
+                    else{
+                        if(@("10.5","10.5.1","10.4.1").Contains($ConfigurationData.ConfigData.Version)){
```

Comment:
Why is it only for this specific versions?

Reply:
From 10.6 onwards, the server comes with a Self Signed Certificate already installed.